dev/ProgressBar/ProgressBarAutomationPeer.cpp

```diff
@@ -31,6 +39,29 @@ hstring ProgressBarAutomationPeer::GetClassNameCore()
     return winrt::hstring_name_of<winrt::ProgressBar>();
 }
 
+winrt::hstring ProgressBarAutomationPeer::GetNameCore()
+{
+    //Check to see if the item has a defined AutomationProperties.Name
+    winrt::hstring name = __super::GetNameCore();
```

Comment:
what do we get as name if AutomationProperties.Name is not set ?

Reply:
We get an `winrt::hstring` whose` m_handle` is `0x0`. This results in no problems when using/returning.